README.md

```diff
@@ -38,6 +38,19 @@ This plugin remains under development and contributions are very welcome, please
 pip install napari-animation
 ```
 
+> [!WARNING]
+> `napari-animation` uses `ffmpeg` to export animations. If you are using a macOS arm64 computer (e.g. M1, M2 processor)
```

Comment:
You might like to also include "Apple Silicon" as a keyword here. Some users may not recognize synonyms, so it's helpful to include.